tests/dataQualityHelpers.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable global-require */
+let locations = [];
+let timeseries = [];
+try {
+  locations = require('../dist/locations.json');
```

Comment:
These will be generated dynamically, so might want to chuck some eslint ignores in here.